helper/ldaputil/config_test.go

```diff
@@ -0,0 +1,74 @@
+package ldaputil
+
+import "testing"
+
+func TestCertificateValidation(t *testing.T) {
```

Comment:
This test is new, brought over from the plugin code since it was no longer needed there.